### PR TITLE
Chore: remove enterprise symbols service

### PR DESCRIPTION
**Pull request by camdencheek**
Now that all code is enterprise licensed, we don't need the service splits. This makes the normal `cmd/symbols` command behave like the old `enterprise/cmd/symbols` command and removes the `enterprise/cmd/symbols` package.

## Test plan

Depending a bit on CI to test the builds, but I ran a few symbols-related smoketests with `sg` locally. Fairly low risk because there wasn't much difference between the enterprise and the OSS versions of the binary.

### CODE REVIEW

**Comment by camdencheek** [comment]
@eseliger, tagging you for a review since you seem to care about this stuff too 😄

File: cmd/symbols/main.go [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
 	"github.com/sourcegraph/sourcegraph/internal/sanitycheck"
+	"github.com/sourcegraph/sourcegraph/internal/service/svcmain"
 )
 
+var config = svcmain.Config{}
+
 func main() {
 	sanitycheck.Pass()
-	osscmd.SingleServiceMainOSS(shared.Service)
+	svcmain.SingleServiceMain(shared.Service, config)
```
Comment (eseliger) [inline cmd/symbols/main.go:15]: I think we don't use this consistently yet - wonder if that causes issues some day 😬

File: cmd/symbols/shared/service.go [hunk 2]
```diff
@@ -15,11 +15,12 @@ func (svc) Name() string { return "symbols" }
...
 func (svc) Configure() (env.Config, []debugserver.Endpoint) {
 	LoadConfig()
-	return nil, []debugserver.Endpoint{GRPCWebUIDebugEndpoint()}
+	config := loadRockskipConfig(env.BaseConfig{}, CtagsConfig, RepositoryFetcherConfig)
+	return &config, []debugserver.Endpoint{GRPCWebUIDebugEndpoint()}
 }
 
 func (svc) Start(ctx context.Context, observationCtx *observation.Context, ready service.ReadyFunc, config env.Config) error {
-	return Main(ctx, observationCtx, ready, SetupSqlite)
+	return Main(ctx, observationCtx, ready, CreateSetup(*config.(*rockskipConfig)))
```
Comment (camdencheek) [inline cmd/symbols/shared/service.go:23]: This is basically just moving `enterprise/cmd/symbols/shared/service.go` into this file. Basically, it just allows rockskip to be enabled.

File: enterprise/cmd/symbols/BUILD.bazel [hunk 3]
```diff
@@ -1,76 +0,0 @@
...
-        "exclusive",
-        "requires-network",
-    ],
-)
-
-oci_push(
-    name = "candidate_push",
-    image = ":image",
-    repository = image_repository("symbols"),
-)
```
Comment (eseliger) [inline enterprise/cmd/symbols/BUILD.bazel:76]: i think you need to at least copy this one over, maybe some other things too.

**Review by eseliger — APPROVED** [review]

File: enterprise/cmd/symbols/BUILD.bazel (hunk 3 above)
Comment (camdencheek) [reply enterprise/cmd/symbols/BUILD.bazel:76]: Oh, did we not push our OSS images? Interesting 🤔 But yeah, good catch, thanks!

Comment (eseliger) [reply enterprise/cmd/symbols/BUILD.bazel:76]: nope, never 😬 since bazel is actually the first time we had even working build scripts for those images

**Pull request merged**